config/mise.php

```diff
@@ -6,4 +6,20 @@
         'laravel-local-developer-tooling' => \App\Recipes\LaravelLocalDeveloperTooling::class,
         'testing' => \App\Recipes\Testing::class,
     ],
+
+    'steps' => [
```

Comment:
It would be nice to intuit this from the class.

Either intuit it from the classname.. e.g. 

`App\Steps\Laravel\InstallHorizon` automatically translates to `laravel/install-horizon` 

.. and/or allow it to be defined in the class for the step.